### PR TITLE
zh_CN version of data_vis_makie_layouts.md

### PR DESCRIPTION
**Pull request by guixinliu**
Hi,  @findmyway 。

我又翻译了data_vis_makie_layouts.md，请您 Review。

### CODE REVIEW

**Comment by findmyway** [comment]
Hi @rikhuijzer , 

Do you know why the CI failed with this PR? It seems quite strange to me and we really don't know how to fix it. The main branch seems to be built successfully.

File: contents/data_vis_makie_attributes.md [hunk 1]
```diff
@@ -146,20 +147,20 @@ s = """
...
+What are all the possible plotting functions available in `Makie.jl`?
+To answer this question, a _cheat sheet_ is shown in @fig:cheat_sheet_cairomakie.
+These work especially well with `CairoMakie.jl` backend.
 
 ![Plotting functions: Cheat Sheet. Output given by Cairomakie.](images/makiePlottingFunctionsHide.png){#fig:cheat_sheet_cairomakie}
 
-@fig:cheat_sheet_glmakie 展示了 `GLMakie.jl` 的_cheat sheet_ ，这些函数支持绘制大多数 3D 图。
-这些将在 @sec:glmakie 进一步讨论。
+For completeness, in @fig:cheat_sheet_glmakie, we show the corresponding functions _cheat sheet_ for `GLMakie.jl`, which supports mostly 3D plots.
+Those will be explained in detail in @sec:glmakie.
```
Comment (findmyway) [inline contents/data_vis_makie_attributes.md]:
```suggestion
Those will be explained in detail in @sec:glmakie .
```

File: contents/data_vis_makie_layouts.md [hunk 2]
```diff
@@ -113,75 +113,75 @@ using Dates
...
+从上图也可以看到 `colsize!` 和 `rowsize!` 如何作用于不同的行和列。
+可以向函数传递一个数字而不是 `Auto()`，但那会固定所有的设置。
+另外， 在定义 `Axis` 时也可以设定 `height` 或 `width`，例如 `Axis(fig, heigth=50)` 将会固定轴的高度。
 
-### Nested `Axis` (_subplots_)
+### 嵌套 `Axis` (_subplots_)
 
-It is also possible to define a set of `Axis` (_subplots_) explicitly, and use it to build a main figure with several rows and columns.
-For instance, the following its a "complicated" arrangement of `Axis`:
+准确定义一组 `Axis` (_subplots_) 是可行的。 因此可以使用一组 `Axis` 构造具有多行多列的图。
```
Comment (findmyway) [inline contents/data_vis_makie_layouts.md]:
```suggestion
精准定义一组 `Axis` (_subplots_) 也是可行的， 可以使用一组 `Axis` 构造具有多行多列的图。
```

File: contents/data_vis_makie_layouts.md [hunk 3]
```diff
@@ -113,75 +113,75 @@ using Dates
...
-Now, using `rowgap!` or `colsize!` over each group is possible and `rowsize!, colsize!` can also be applied to the set of `GridLayout()`s.
+现在，对每一组使用 `rowgap!` 或 `colsize!` 将是可行的，并且 `rowsize!, colsize!` 也能够应用于 `GridLayout()`。
 
-### Inset plots
+### 插图
 
-Currently, doing `inset` plots is a little bit tricky.
-Here, we show two possible ways of doing it by initially defining auxiliary functions.
-The first one is by doing a `BBox`, which lives in the whole `Figure` space:
+目前，绘制 `插图` 是一项棘手的工作。
```
Comment (findmyway) [inline contents/data_vis_makie_layouts.md]:
```suggestion
目前，绘制 `inset` 是一项棘手的工作。
```

**Comment by rikhuijzer** [comment]
Ah nice. Yes seems you discovered that things inside backticks cannot be Unicode 😞 I don’t know how to fix that

**Pull request merged**